introduction.md

```diff
@@ -0,0 +1 @@
+http://campus.codeschool.com/courses/front-end-foundations/contents
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.